terraform/files/bin/bootstrap.sh

```diff
@@ -87,6 +87,7 @@ install_helm.sh
 deploy_cluster_api.sh
 install_k9s.sh
 get_capi_helm.sh
+install_kubectx.sh
```

Comment:
Minor: Would be great (sooner is better) to introduce the `Release-Notes-R6.md` doc and fill it appropriately.